src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,215 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
```

Comment:
According to the checklist, the class `Node<T>` should be an inner class of `MyLinkedList<T>`. It should be located at the end of the class, which is correctly done in your code.